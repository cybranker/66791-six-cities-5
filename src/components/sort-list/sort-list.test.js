import React from "react";
import renderer from "react-test-renderer";
import SortList from "./sort-list";

const noop = () => {};

test(`Should SortList render correctly`, () => {
  const tree = renderer
    .create(<SortList
      isOpenSortList={false}
      sortType={`popular`}
      toggleSortList={noop}
      changeSortType={noop}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
