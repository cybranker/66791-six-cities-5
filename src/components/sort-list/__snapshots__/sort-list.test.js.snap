// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should SortList render correctly 1`] = `
<ul
  className="places__options places__options--custom false"
  onClick={[Function]}
>
  <li
    className="places__option places__option--active"
    onClick={[Function]}
    tabIndex="0"
  >
    Popular
  </li>
  <li
    className="places__option false"
    onClick={[Function]}
    tabIndex="0"
  >
    Price: low to high
  </li>
  <li
    className="places__option false"
    onClick={[Function]}
    tabIndex="0"
  >
    Price: high to low
  </li>
  <li
    className="places__option false"
    onClick={[Function]}
    tabIndex="0"
  >
    Top rated first
  </li>
</ul>
`;
